tests/fixtures/incl2.h

```diff
@@ -0,0 +1,2 @@
+#define CONST2 123
+#define CONST3 777
```

Comment:
there is no newline at the end of the file - diff unhappy.

Reply:
Fixed.